sapi/fpm/fpm/fpm_status.c

```diff
@@ -398,21 +398,18 @@ int fpm_status_handle_request(void) /* {{{ */
 				"# HELP phpfpm_up Could pool %s using a %s PM on PHP-FPM be reached?\n"
 				"# TYPE phpfpm_up gauge\n"
 				"phpfpm_up 1\n"
-				"# HELP phpfpm_start_time When FPM has started.\n"
-				"# TYPE phpfpm_start_time gauge\n"
-				"phpfpm_start_time %lld\n"
-				"# HELP phpfpm_start_since_total The number of seconds since FPM has started.\n"
-				"# TYPE phpfpm_start_since_total counter\n"
-				"phpfpm_start_since_total %lu\n"
-				"# HELP phpfpm_accepted_connections_total The number of requests accepted by the pool.\n"
-				"# TYPE phpfpm_accepted_connections_total counter\n"
-				"phpfpm_accepted_connections_total %lu\n"
+				"# HELP phpfpm_start_since The number of seconds since FPM has started.\n"
+				"# TYPE phpfpm_start_since counter\n"
+				"phpfpm_start_since %lu\n"
+				"# HELP phpfpm_accepted_connections The number of requests accepted by the pool.\n"
+				"# TYPE phpfpm_accepted_connections counter\n"
+				"phpfpm_accepted_connections %lu\n"
 				"# HELP phpfpm_listen_queue The number of requests in the queue of pending connections.\n"
 				"# TYPE phpfpm_listen_queue gauge\n"
 				"phpfpm_listen_queue %d\n"
-				"# HELP phpfpm_max_listen_queue_total The maximum number of requests in the queue of pending connections since FPM has started.\n"
-				"# TYPE phpfpm_max_listen_queue_total counter\n"
-				"phpfpm_max_listen_queue_total %d\n"
+				"# HELP phpfpm_max_listen_queue The maximum number of requests in the queue of pending connections since FPM has started.\n"
+				"# TYPE phpfpm_max_listen_queue counter\n"
+				"phpfpm_max_listen_queue %d\n"
```

Comment:
Hey @bukka I get the reasoning for this change, the reason I went for these names instead of matching them with that exporter is to match the Prometheus naming conversion: https://prometheus.io/docs/practices/naming/ 

The main question that is going around in my mind right now, is do we want to match up with that exporter, or set our own naming following the Prometheus naming convention?

Reply:
Well you agreed about the sync: https://github.com/php/php-src/pull/5723#issuecomment-804847794 so I did it that way. I think there's some value to have a compatibility mode with the exporter as it would be make the migration pretty easy. Although after reading the conventions I feel the right way would be to following them. Maybe we should later introduce some kind of versioning with a format using correct conventions and labels. Now it's too late to change anything as beta1 has been just released.

Reply:
> Well you agreed about the sync: [#5723 (comment)](https://github.com/php/php-src/pull/5723#issuecomment-804847794) so I did it that way. 

Correct, and I'm on the fence about it now. Some of them actually make sense to me, like `phpfpm_max_active_processes`, but might not follow that convention.

> I think there's some value to have a compatibility mode with the exporter as it makes the migration pretty easy. Although after reading the conventions I feel the right way would be to follow them. Maybe we should later introduce some kind of versioning with a format using correct conventions and labels. 

How would versioning work? An additional query field to version that? 

> Now it's too late to change anything as beta1 has been just released.

*shrug* it's not a very big deal, just think it would be nice if we did it perfectly :sunglasses: . Having this out there is a massive win already :tada: .

Reply:
> How would versioning work? An additional query field to version that?

I was thinking about making more query based with the same keys and different values and not differentiated by keys. For example something like `format=openmetrics&version=2`.

Reply:
We would of course keep the old queries working for quite some time so it doesn't break between the versions.

Reply:
That would work, assuming that will come in PHP 8.2 then?

Reply:
Yeah hopefully :)

Reply:
Well then let me pot an reminder for that in my calendar :D